### PR TITLE
Change country codes to alpha2code -- addresses issue #21

### LINKED ISSUE #21: Consider renaming "country" field
Possible alternatives include "economy", "ISO3166.1a2 field", et cetera.

### PR DESCRIPTION
**Pull request by wkumari**

### CODE REVIEW

File: draft.xml [hunk 1]
```diff
@@ -244,11 +244,13 @@
...
 
-          <section title="Country">
-            <t>OPTIONAL. The country field, if non-empty, MUST be a 2 letter
+          <section title="Country / alpha2code">
+            <t>OPTIONAL. The alpha2code field, if non-empty, MUST be a 2 letter
             ISO country code conforming to ISO 3166-1 alpha 2 <xref
             target="ISO.3166.1alpha2"/>. Parsers SHOULD treat this field
-            case-insensitively.</t>
+            case-insensitively. Earlier versions of this document called this
+            field "country", and it may still be referred to as this in
```
Comment (ekline) [inline draft.xml]: Suggest: "referred to as such in"  (s/this/such/)

File: draft.xml [hunk 2]
```diff
@@ -244,11 +244,13 @@
             "2001:db8::1" and "2001:db8::/32" for IPv6.</t>
           </section>
 
-          <section title="Country">
-            <t>OPTIONAL. The country field, if non-empty, MUST be a 2 letter
+          <section title="Country / alpha2code">
```
Comment (ekline) [inline draft.xml]: Suggest: title="Alpha2code [previously: country]"?

**Review by ekline — APPROVED** [review]
Thank you!

(can I really edit this PR in flight?)

File: draft.xml (hunk 2 above)
Comment (wkumari) [reply draft.xml]: Ugh, I dislike this, but sure, whatever...

**Pull request merged**